### PR TITLE
WRO-11451: Created "perf" samples for agate components

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Agate repo did not have perf samples.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Created perf samples, similar to Sandstone ones.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-11451

### CODE REVIEW

File: samples/perf-virtuallist-native-with-different-item-size/src/App.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+	const onSelect = useCallback(({selected}) => {
+		setIndex(selected);
+	}, []);
+
+	const View = views[index];
+
+	return (
+		<div {...props}>
+			<Dropdown
```
Comment (daniel-stoian-lgp) [inline samples/perf-virtuallist-native-with-different-item-size/src/App.js:34]: this dropdown doesn't get the default value on application load. can you check please?
image

**Comment by stanca-pop-lgp** [comment]
@daniel-stoian-lgp thank you for you review! I pushed the fixes.

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by jeonghee27 — APPROVED** [review]
LGTM
@daniel-stoian-lgp 
Thank you for your review.

**Pull request merged**